### PR TITLE
Document Quay security on container images

### PR DESCRIPTION
**Pull request by fridex**
## This introduces a breaking change

- [x] No

### CODE REVIEW

File: docs/source/security.rst [hunk 1]
```diff
@@ -72,6 +72,16 @@ If you are interested, follow `scorecards checks available
...
 
+Quay security scans of container image
+######################################
+
+Another source of security information is security data from Quay [#quay]_
+security scanners. Quay scans container images for security vulnerabilities.
+Information about vulnerabilities in container images is automatically
+aggregated by Thoth's background data aggregation logic for container images
+provided by Thoth team. By using these container images, users get fine-grained
+control on vulnerabilities present also on container image level.
```
Comment (KPostOffice) [inline docs/source/security.rst:83]:
```suggestion
control on vulnerabilities present on the container image level in addition to their software stacks.
```

**Review by KPostOffice — COMMENTED** [review]
One small comment.

**Comment by fridex** [comment]
> One small comment.

Thanks, applied 👍🏻

**Pull request merged**